assets/js/components/legacy-notifications/dashboard-modules-alerts.js

```diff
@@ -74,7 +74,7 @@ class DashboardModulesAlerts extends Component {
 						title={ notification.title || '' }
 						description={ notification.description || '' }
 						blockData={ notification.blockData || [] }
-						winImage={ notification.winImage ? `${ global._googlesitekitLegacyData.admin.assetsRoot }images/${ notification.winImage }` : '' }
+						WinImageSVG={ notification.WinImageSVG }
```

Comment:
This won't work because `notification` comes from an API response here. I think this would be the same as the change to `Alert` where it would get `SmallSunSVG`.

Reply:
Updated in https://github.com/google/site-kit-wp/pull/2834/commits/2502ef0c930661df6ff9359f7342328d4124e6be